libnmstate/nm/device.py

```diff
@@ -324,3 +324,13 @@ def get_device_common_info(dev):
         'type_name': dev.get_type_description(),
         'state': dev.get_state(),
     }
+
+
+def delete_device_inactive_profiles(ifname):
```

Comment:
This function has nothing to do with a device.
If you wanted to delete all device profiles, you can use the existing `device` functions.

I think you need to place this under the `connection` module and name it accordingly.
Something like:
```
for con in list_connections_by_ifname(nmclient.client()):
    con.delete()
```
`list_connections_by_ifname` should return in this case `ConnectionProfile` objects.

Reply:
Done, thanks!